doc/compiled.json

```diff
@@ -26575,6 +26622,207 @@
+          "400": {
+            "description": "Bad Request",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "message": {
+                      "type": "string"
+                    }
+                  },
+                  "example": {
+                    "message": "Key <parent_key_code> is not a parent key"
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "post": {
+        "summary": "Link child keys to a parent key",
+        "description": "Creates links between a given parent key and one or more child keys.",
+        "operationId": "linked_keys/create",
+        "tags": [
+          "Linked Keys"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/X-PhraseApp-OTP"
+          },
+          {
+            "$ref": "#/components/parameters/account_id"
+          }
+        ],
+        "requestBody": {
+          "required": true,
+          "content": {
+            "application/json": {
+              "schema": {
+                "required": [
+                  "child_key_ids"
+                ],
+                "type": "object",
+                "title": "linked_keys/create/parameters",
+                "properties": {
+                  "child_key_ids": {
+                    "description": "The IDs of the child keys to link to the parent key. Can be left empty, to only mark the given translation-key as parent",
+                    "type": "array",
+                    "example": [
+                      "child_key_id1",
+                      "child_key_id2"
+                    ],
+                    "items": {
+                      "type": "string"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "responses": {
+          "201": {
+            "description": "Created",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/linked_key_reference"
+                }
+              }
+            }
+          },
+          "422": {
+            "$ref": "#/components/responses/422"
+          }
+        }
+      }
+    },
+    "/projects/{project_id}/keys/{id}/key_links/{child_key_id}": {
+      "delete": {
+        "summary": "Unlink a child key from a parent key",
+        "description": "Unlinks a single child key from a given parent key.",
+        "operationId": "linked_keys/destroy",
+        "tags": [
+          "Linked Keys"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/X-PhraseApp-OTP"
+          },
+          {
+            "$ref": "#/components/parameters/account_id"
+          },
+          {
+            "in": "path",
+            "name": "linked_key_id",
```

Comment:
I think we renamed this to `child_key_id`?

Reply:
:pray: